tools/gromacs/test-data/posre_cl.itp

```diff
@@ -1,12 +1,6 @@
-; position restraints for CL of LYSOZYME in water
+; position restraints for CL of �-�y�U
```

Comment:
uh, this looks strange?

Reply:
I think it's a GROMACS bug. I resorted to this to fix it, since Galaxy seems to get really confused by it: https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/pull/72/commits/f426e6d6b7476b495143dca789134b0239d4fdb5#diff-a4db2c2bf23fc0cec5379f8b9d9a92d6R15